File: C++11/백준/Bronze/21197. Stopwatch/Stopwatch.cc
#include <iostream>
using namespace std;

int main(){
    int a,b,c,s=0;
    cin >> a;
    if (a%2) cout << "still running";
    else{
        for (int i=0;i<a/2;i++){
            cin >> b>>c;
            s+=c-b;
        }
        cout << s;
    }
}

File: C#/백준/Bronze/21197. Stopwatch/Stopwatch.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int s = 0;
            if (a%2==1){
                Console.WriteLine("still running");
            }
            else{
                for (int i=0;i<a/2;i++){
                    int b = int.Parse(Console.ReadLine());
                    int c = int.Parse(Console.ReadLine());
                    s+=c-b;
                }
                Console.WriteLine(s);
            }
        }
    }
}